addons/mail/static/src/models/composer/composer.js

```diff
@@ -176,7 +176,8 @@ function factory(dependencies) {
             const recipients = [...this.mentionedPartners];
             if (this.thread) {
                 for (const recipient of this.thread.suggestedRecipientInfoList) {
-                    if (recipient.isSelected) {
+                    // If there is no partner we don't want to push the recipient
```

Comment:
Either no caps at the start, or have a dot at the end.
And actually comment is useless because it is saying exactly what the code does. Either you explain a functional need (the "why", eg. message post expecting a list of partners as recipients, or the modal to create partner for missing one) or there is no need for the comment at all.